presto-main/src/test/java/com/facebook/presto/type/TestRowOperators.java

```diff
@@ -104,6 +119,9 @@ public void testRowCast()
         assertFunction("cast(test_row(2, cast(null as double)) as row(aa bigint, bb double)).bb", DOUBLE, null);
         assertFunction("cast(test_row(2, 'test_str') as row(aa bigint, bb varchar)).bb", VARCHAR, "test_str");
 
+        // Should fail for ambiguity
```

Comment:
I think this one is ok, I will just remove, since the column names in a table are not case sensitive either.
Or better, I can make it expect an exception like with the hyperloglog test here.